crates/uv/tests/it/lock.rs

```diff
@@ -3765,6 +3765,197 @@ fn lock_conflicting_group_basic() -> Result<()> {
     Ok(())
 }
 
+/// This tests a case of specifying conflicting groups, where some of the conflicts are enabled by
+/// default.
+#[test]
+fn lock_conflicting_group_default() -> Result<()> {
+    let context = TestContext::new("3.12");
+
+    // Tell uv about the conflicting groups, which forces it to resolve each in
+    // their own fork.
+    let pyproject_toml = context.temp_dir.child("pyproject.toml");
+    pyproject_toml.write_str(
+        r#"
+        [project]
+        name = "project"
+        version = "0.1.0"
+        description = "Add your description here"
+        requires-python = ">=3.12"
+
+        [tool.uv]
+        conflicts = [
+            [
+              { group = "project1" },
+              { group = "project2" },
```

Comment:
As in the other pull request, can we name these differently please?

Reply:
This series of tests all use this style, I guess I'd rather do it in a single PR and change all of them.

Reply:
Yeah no problem.